### DISCUSSION TITLE
Is there a way to "flag" Whisky as game to use gamemode in Sonoma?

### DISCUSSION THREAD

**Discussion by lzanchin** [Whisky-App/Whisky#182] [Ideas]
I was just wondering if there is a way to tag/flag or somehow, by code, make the OS see the app as game and enable gamemode for it to benefit for the so called "improved performance" and latency reduction on bluetooth?

**Comment by IsaacMarovitz** [comment]
Currently, no

**Comment by abnerfreitas** [comment]
Any updates on this? Is it impossible (probably bc its a Apple bs) or its just hard to implement?
My games runs great on Whisky by default, i just wanted the low-latency bluetooth :P
